multiqc/modules/fastp/fastp.py

```diff
@@ -232,6 +232,8 @@ def parse_fastp_log(self, f):
             ) * 100.0
         except KeyError:
             log.debug("Could not calculate 'pct_adapter': {}".format(f["fn"]))
+        except ZeroDivisionError:
+            log.warning("FastQ input file has zero raw reads")
```

Comment:
Could you please log the sample name here?

Reply:
In fact I'd be inclined to just merge this with the catch / log message above.. What do you think?

Reply:
I think it would be best to either report once per sample or report the sample in the message.  What is your opinion of adding a `pass` instead of logging sample?

Using the data from #1499 and `MultiQC` build from 2dd31c1 results in the following:

```bash
  /// MultiQC 🔍 | v1.12.dev0

|           multiqc | Search path : /home/johnny/data/testing/multiqc_fastp_bug/PCRblank.fastp.json
|         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 1/1
|             fastp | FastQ input file has zero raw reads
|             fastp | FastQ input file has zero raw reads
|             fastp | Found 1 reports
|           multiqc | Compressing plot data
|           multiqc | Report      : report/multiqc_report.html
|           multiqc | Data        : report/multiqc_data
|           multiqc | MultiQC complete
```
which I think is confusing given that it is one sample.

Reply:
the sample does make it into the report.

Reply:
Sorry for the delay. Yup, I think that we're saying the same thing here - as it is the error message is a bit confusing (repeated, sample is still there).

So I was suggesting that we log the same name (no longer repeated) or just merge with the statement above (debug so not in the main console output unless verbose, also specifies what is going wrong and the sample name). I think I'll go for the latter if that's ok.

Reply:
Tweaked in fe471c6